controllers/auth/register.js

```diff
@@ -0,0 +1,30 @@
+const gravatar = require("gravatar");
+const User = require("../../models/user");
+const bcrypt = require("bcrypt");
+const {HttpError} = require("../../helpers");
+require("dotenv").config();
+
+const register = async (req, res, next) => {
+    try {
+     const { email, password } = req.body;
+    const user = await User.findOne({email});
+    if(user) {
+        throw HttpError(409, "Email in use");
+    }
+
+    const hashPassword = await bcrypt.hash(password, 10);
+    const avatarURL = gravatar.url(email);
+    console.log(avatarURL)
```

Comment:
```suggestion
    console.log(avatarURL)
```
видаліть зайві console.log